x-pack/plugins/apm/public/components/shared/charts/TransactionCharts/index.tsx

```diff
@@ -96,18 +96,19 @@ export class TransactionCharts extends Component<TransactionChartProps> {
   };
 
   public renderMLHeader(hasValidMlLicense: boolean | undefined) {
-    const { hasMLJob } = this.props;
-    if (!hasValidMlLicense || !hasMLJob) {
+    const {
+      hasMLJob,
+      charts: { mlJobId },
+    } = this.props;
+    if (!hasValidMlLicense || !hasMLJob || !mlJobId) {
```

Comment:
What's the difference between `!hasMLJob` and `!mlJobId` ?

Reply:
There's really no difference other than types, but the the bang coerces either type to a boolean, so i can safely remove `hasMLJob`.